### PR TITLE
Update kubectl to the stable version

### LINKED ISSUE #54762: kubectl need to update
**Is this a BUG REPORT or FEATURE REQUEST?**:
 kubectl need to update
> Uncomment only one, leave it on its own line: 
>
> /kind bug
> /kind feature

**What happened**:

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:
Update kubectl to the stable version 1.8.1
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
fixes #54762
**Special notes for your reviewer**:

**Release note**:

```
Update kubectl in the addon-manager to v1.8.4
```

### CODE REVIEW

**Comment by roberthbailey** [comment]
Please squash your commits and add a release note. 

/cc @wojtek-t

**Comment by zouyee** [comment]
@roberthbailey has fixed it, thank you!

**Comment by zouyee** [comment]
@roberthbailey  would you like to approve it? thanks!

**Comment by roberthbailey** [comment]
I'd like @wojtek-t to take a look first

**Comment by wojtek-t** [comment]
I'm fine with that.

/lgtm

**Comment by zouyee** [comment]
gentle ping @roberthbailey

**Comment by stewart-yu** [comment]
I have a question why not update to v1.8.2 now? @zouyee

**Comment by roberthbailey** [comment]
Are there any changes that affect kubectl in 1.8.2? I don't see anything pertinent listed in the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.8.md#downloads-for-v182).

**Review by mariida — APPROVED** [review]

**Comment by zouyee** [comment]
ping @wojtek-t

**Comment by cblecker** [comment]
PR contains merge commit. Please remove it.
/hold

**Comment by roberthbailey** [comment]
@zouyee - this seems like a good change to get it for 1.9 (to keep the client within supported version skew). Can you remove the merge commit (as suggested by @cblecker above) so that we can consider it during the code slush?

**Comment by wojtek-t** [comment]
> @zouyee - this seems like a good change to get it for 1.9 (to keep the client within supported version skew). Can you remove the merge commit (as suggested by @cblecker above) so that we can consider it during the code slush?

+1

**Comment by zouyee** [comment]
I was caught in work and missed the PR.

**Comment by cblecker** [comment]
@roberthbailey @wojtek-t Over to you for milestone/labels to allow this to merge.

File: cluster/addons/addon-manager/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+### Version 8.1  (Sun October 22 2017 zou nengren @zouyee)
```
Comment (roberthbailey) [inline cluster/addons/addon-manager/CHANGELOG.md]: Can you update the date here and bump the version from 1.8.1 --> 1.8.4?

Comment (zouyee) [reply cluster/addons/addon-manager/CHANGELOG.md]: got it!

**Comment by cblecker** [comment]
@roberthbailey If this should get into 1.9, it needs the milestone added. Are you able to do this?

**Comment by zouyee** [comment]
kops-aws error
```
It appears that the test is looking for cat /proc/net/ip_conntrack, but on 4.9 I have /proc/net/nf_conntrack instead
```
releated issue https://github.com/kubernetes/kubernetes/issues/56763

**Comment by x13n** [comment]
Is addon manager version going to be synced with kubectl version from now on?

**Pull request merged**